Userland/Applications/KeyboardSettings/Keyboard.gml

```diff
@@ -124,4 +124,18 @@
             name: "num_lock_checkbox"
         }
     }
+
+    @GUI::GroupBox {
+        title: "Caps Lock"
+        fixed_height: 60
+        layout: @GUI::HorizontalBoxLayout {
+            margins: [16, 8, 8]
+            spacing: 16
+        }
+
+        @GUI::CheckBox {
+            text: "Remap Caps Lock to Ctrl"
```

Comment:
‘Remapʼ is probably a good one; yet if itʼs more of exchange/switch etc., then maybe it could be better to use something like that for clarity…? 🤔

(here and in other places; it might be that kernel variable is a misnomer but thatʼs definitely out of scope)

Reply:
Yes, though it's my understanding that the kernel variable replaces Caps Lock with an "additional" Ctrl, but the behavior of the physical Ctrl key is unchanged. I fear that exchange/switch imply that the keys will be swapped. Perhaps something like "Use Caps Lock as an additional Ctrl" would work?